### PR TITLE
Fixed bug while presenting song

### PR DESCRIPTION
**Pull request by madasamys**

### CODE REVIEW

File: worshipsongs/worshipsongs/SongWithVideoViewController.swift [hunk 1]
```diff
@@ -611,9 +606,10 @@ extension SongWithVideoViewController: UIGestureRecognizerDelegate {
     
     internal func onCellViewLongPress(_ longPressGesture: UILongPressGestureRecognizer) {
         
-        if previousButton.isHidden && previousButton.isHidden {
+        if previousButton.isHidden && nextButton.isHidden {
```
Comment (VigneshPalanisamy) [inline worshipsongs/worshipsongs/SongWithVideoViewController.swift]: in project mode while in first lyrics previous button wont visible and in last lyrics next button won't visible at that time long press will execute!

File: worshipsongs/worshipsongs/SongWithVideoViewController.swift [hunk 2]
```diff
@@ -648,6 +644,7 @@ extension SongWithVideoViewController: UIGestureRecognizerDelegate {
         activityVC.excludedActivityTypes = [UIActivityType.airDrop, UIActivityType.postToWeibo, UIActivityType.postToVimeo, UIActivityType.postToTencentWeibo, UIActivityType.postToFlickr, UIActivityType.assignToContact, UIActivityType.addToReadingList, UIActivityType.copyToPasteboard, UIActivityType.postToFacebook, UIActivityType.saveToCameraRoll, UIActivityType.print, UIActivityType.openInIBooks, UIActivityType(rawValue: "Reminders")]
         self.present(activityVC, animated: true, completion: { () -> Void in
             self.tableView.deselectRow(at: indexPath, animated: true)
+            self.tableView.allowsSelection = false
```
Comment (VigneshPalanisamy) [inline worshipsongs/worshipsongs/SongWithVideoViewController.swift]: will setting allowsSelection works without tableview data reload?

File: worshipsongs/worshipsongs/SongWithVideoViewController.swift (hunk 1 above)
Comment (VigneshPalanisamy) [inline worshipsongs/worshipsongs/SongWithVideoViewController.swift]: no need the condition.

Comment (madasamys) [inline worshipsongs/worshipsongs/SongWithVideoViewController.swift]: It should be need. And need to  check either floating or presenting button is visible.

**Pull request merged**